questionnaireForm.php

```diff
@@ -0,0 +1,113 @@
+<?php
+include('config/init.php');
+echologgedInHeader("questionnaire");
+$user = getUser($_SESSION['accountId']); 
+$accountId = $user['accountId'];
+
+if(isset($_REQUEST['NewQuestionnaireData'])){
+   insertNewQuestionnaireData(
+       $accountId,
+       $_REQUEST['questionnaireBeverage'],
+       $_REQUEST['questionnaireBeverageOther'], 
+       $_REQUEST['beverageSpecific'], 
+       $_REQUEST['howMuchBeverage'], 
+       $_REQUEST['howMuchWater'], 
+       $_REQUEST['sleep'], 
+       $_REQUEST['stress'], 
+       $_REQUEST['breakfast'],
+       $_REQUEST['lunch'],
+       $_REQUEST['dinner'], 
+       $_REQUEST['exercise']
+       
```

Comment:
You'll probably want to redirect the user to some kind of confirmation page. OTherwise they might think their answer wasn't saved